code/game/objects/items/grenades/discogrenade.dm

```diff
@@ -48,7 +48,6 @@
 	var/power = 3
 
 /obj/item/grenade/discogrenade/subgrenade/Initialize(mapload, duplicate = FALSE)
-	. = ..()
```

Comment:
Every atom needs to call parent in initialize. This should not have been removed.